integration_testing/features/super-admin/super-admin-windows-installer-translated-strings.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Super admin can see the Windows installer translated strings
+    Super admin can see the Windows installer translated strings during the Kolibri Windows install/uninstall
+
+    Background:
+        Given That the Windows language setting is in <language>
+        Given I am using Windows environment
+        And I have the Kolibri Windows installer
+
+# List of translation from Spanish to English
+# "Control de cuentas usuario" > "User Account Control"
+# "Seleccionar idioma de Configuración" > "Select Setup Language"
+# "Kolibri configuración" > "Kolibri Setup Wizard"
+# "Iniciar Kolibri" > "Launch Kolibri"
+# "Desinstalar un programa" > "Uninstall a Program"
+# "Aceptar" > "Ok"
+# "Sí" > "yes"
+# "Siguiente" > "Install"
+# "Finalizar" > "Finish"
+# "Panel de control" > "Control Panel"
```

Comment:
Thanks for putting the translated strings here, should help a lot during testing.